Yosemite/Yosemite/Stores/StatsStoreV4.swift

```diff
@@ -285,6 +296,100 @@ extension StatsStoreV4 {
     }
 }
 
+
+// MARK: Convert Leaderboard into TopEarnerStats
+//
+private extension StatsStoreV4 {
+
+    /// Converts and stores a top-product `leaderboard` into a `StatsTopEarner`
+    /// Since  a `leaderboard` does not containt  the necesary product information, this method fetches the related product before starting the convertion.
+    ///
+    func convertAndsStoreLeaderboarsIntoTopEarners(siteID: Int64,
```

Comment:
super nit: "convertAndsStore" -> "convertAndStore" and "Leaderboars" -> "Leaderboards"?